core/types/block.go

```diff
@@ -40,14 +40,14 @@ func (b *Block) Serialization(sink *common.ZeroCopySink) {
 	shardIds := make([]uint64, 0, len(b.ShardTxs))
 
 	for id := range b.ShardTxs {
-		shardIds = append(shardIds, id)
+		shardIds = append(shardIds, id.ToUint64())
```

Comment:
also change this shardIds from []uint64 to []common.ShardID ?

Reply:
ok